src/base/_defaults.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Sensible box model
+ */
+*,
+::before,
+::after {
```

Comment:
You're only applying the sensible box model to generated content? I've typically seen this rule written to include a `*` selector, so everything gets it by default.

Reply:
OMG I totally missed the first `*` selector because my brain lumped it in with the comment.

Sorry! My bad. I'll approve now.